content/blog/end-to-end-encryption/index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: End-to-End Encryption in the Browser
+date: '2020-03-21T22:40:32.169Z'
+description: "&nbsp;"
```

Comment:
Why not adding a small description? It will appear on the front page.

Reply:
I feel like the title is good enough. I'm not sure why I need to add another sentence here

Reply:
Fair enough.. then let's just delete the description altogether, and in the upcoming PRs fix the open graph part when it's shared.

Reply:
Can’t delete, it throws a graphql error :(

Reply:
OK.. something for later.